source/introduction.txt

```diff
@@ -278,7 +263,16 @@ services to do things like:
 
 Template apps are working apps you can run and change to experiment with 
 App Services. These apps are a good choice for developers who prefer to learn by 
-experimentation, versus the more description-heavy tutorials above.
+experimentation, versus the tutorials, which provide a deeper, step-by-step 
+process for building apps.
+
+Tutorials
+~~~~~~~~~
+
+Check out our :doc:`tutorials </tutorials>` to learn more about how to build 
+App Services applications. Tutorials start with our :ref:`Template Apps <template-apps>` 
+and walk you through code excerpts that are specific to working
```

Comment:
nit: maybe focus more on the goal (build a new feature) instead of the how (copy paste code excerpts)

> and then walk you through building additional features powered by App Services.

also should we mention sync specifically?

> powered by App Services, including Device Sync and Role-based Permissions.